storage/postgres/tree_storage_test.go

```diff
@@ -24,7 +24,7 @@ type expandTestcase struct {
 	expected string
 }
 
-// This test exists to prevent gometalinter from failing
+// This test exists to prevent golangci-lint from failing
```

Comment:
(preexisting) Be nicer to mention the specific linter that fails rather than its driver: `...to prevent the deadcode linter from failing`